React/wbooks/src/utils/regexs.js

```diff
@@ -0,0 +1,2 @@
+export const emailRegex=/^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```

Comment:
add spaces between `=`

Reply:
Here export a arrow function that receive a value and directly tests that value with this regex